docs/api.md

```diff
@@ -1101,10 +1101,18 @@ Sends message via HTTP protocol
 
 > Must-have: `thing_key`
 
+For SenML:
 ```bash
-curl -s -S -i -X POST -H "Content-Type: application/json" -H "Authorization: Thing <thing_key>" http://localhost/http/messages -d '[{"bn":"some-base-name:","bt":1.276020076001e+09,"bu":"A","bver":5,"n":"voltage","u":"V","v":120.1}, {"n":"current","t":-5,"v":1.2}, {"n":"current","t":-4,"v":1.3}]'
+curl -s -S -i -X POST -H "Authorization: Thing <thing_key>" -H "Content-Type: application/json" "http://localhost/http/messages" -d '[{"bn":"some-base-name:","bt":1.276020076001e+09,"bu":"A","bver":5,"n":"voltage","u":"V","v":120.1}, {"n":"current","t":-5,"v":1.2}, {"n":"current","t":-4,"v":1.3}]'
 ```
 
+
+For JSON:
+```bash
+curl -X POST -H "Authorization: Thing <thing_key>" -H "Content-Type: application/json" "http://localhost/http/messages" -d '[{"n":"temperature","v":21},{"n":"humidity","v":68}]'
```

Comment:
Since SenML it's a JSON this data `[{"n":"temperature","v":21},{"n":"humidity","v":68}]`  is correct but confusing. Use an example that isn't SenML please.